google-cloud-spanner/src/main/java/com/google/cloud/spanner/Json.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner;
+
+import java.io.Serializable;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+public class Json implements Comparable<Json>, Serializable {
```

Comment:
I think we need to add some documentation here. Especially noting that this is a tiny type and no json parsing is done, so if an input string is given it could be completely invalid JSON.